### DISCUSSION TITLE
1.6.1更新日志（伪）

### DISCUSSION THREAD

**Discussion by ShallowDream12321** [share121/inter-knot#1502] [其他]
![image](https://github.com/user-attachments/assets/e63a6956-b60b-41c6-aa9b-c5712c7e7d50)

1、增加刷新按钮
2、增加等级注释
提示：各位可以酌情更新

**Comment by user1-cloud** [comment]
加油，感觉绳网正在慢慢变好

**Comment by cpu644** [comment]
棒的

**Comment by 10kaihua** [comment]
提问：怎么等级注释？F12吗？如何保存等级？

**Comment by fairy-B** [comment]
啊？

**Comment by ShallowDream12321** [comment]
@10kaihua 等级与你的GitHub项目数量有关
进入GitHub主页 
点击右上角加号
创建新项目（需要公开）
然后你就1级了

**Reply by 10kaihua** [reply]
哦豁，多谢解惑，祝绳网越做越好😁

**Reply by 10kaihua** [reply]
是这样吗？为啥不好使
![屏幕截图 2024-08-11 214826](https://github.com/user-attachments/assets/dbc1c9c9-0e57-45dd-921a-dc3693147a92)

**Reply by ShallowDream12321** [reply]
项目名称随便写 然后选公开 最后应该是保存（最底下的 找个翻译插件）然后回绳网就可以看到了
